src/inc/readytorun.h

```diff
@@ -360,4 +360,9 @@ struct READYTORUN_EXCEPTION_CLAUSE
     };  
 };
 
+enum ReadyToRunRuntimeConstants : DWORD
+{
+    READYTORUN_RUNTIME_CONSTANT_PInvokeTransitionFrameSize = 11 * sizeof(void *)
```

Comment:
`sizeof(void *)` is not good here - I think it may even break some cases where cross-compilation works today.

Reply:
It may be easiest to make the size in pointers, and let the place where it is used to multiply it with appropriate pointer size.